src/array_api_stubs/_2022_12/fft.py

```diff
@@ -261,7 +261,7 @@ def rfft(
     Returns
     -------
     out: array
-        an array transformed along the axis (dimension) indicated by ``axis``. The returned array must have a complex-valued floating-point data type determined by :ref:`type-promotion`.
+        an array transformed along the axis (dimension) indicated by ``axis``. The returned array must have a complex floating-point data type whose precision matches the precision of ``x`` (e.g., if ``x`` is ``float64``, then the returned array must have a ``complex128`` data type). The length along the transformed axis is ``n``, if given, or the length of the input along the axis specified by ``axis`` otherwise.
```

Comment:
This also disagrees with [NumPy's `rfft`](https://numpy.org/doc/stable/reference/generated/numpy.fft.rfft.html):

> The truncated or zero-padded input, transformed along the axis indicated by axis, or the last one if axis is not specified. If n is even, the length of the transformed axis is (n/2)+1. If n is odd, the length is (n+1)/2.

Reply:
Yes, would you mind fixing it Aaron?